eldritch/encounters.py

```diff
@@ -252,11 +252,134 @@ def Graveyard6(char):
   return events.Sequence([check, cond], char)
+def Society5(char):
+  return events.Loss(char, {"sanity": 1})
+def Society6(char):
+  # TODO: spend gate trophy
+  prereq = events.AttributePrerequisite(char, "dollars", 1, "at least") # TODO: actually a gate trophy
+  # TODO: prerquisite on ally
+  ally = events.Draw(char, "allies", "Old Professor")
+  events.PassFail(char, prereq, ally, events.Nothing())
+def Society7(char):
+  move = events.ForceMovement(char, "Cave")
+  encounter = events.Encounter(char, "Cave", 2)
+  cave = events.Sequence([move, encounter], char)
+  return events.BinaryChoice(char, "Go with Cindy?", "Yes", "No", cave, events.Nothing())
+
+def House1(char):
+  luck = events. Check(char, "luck", 0)
+  move_dream = events.ForceMovement(char, "Dreamlands1")
+  # TODO: don't hard-code.
+  enc_dream = events.GateEncounter(char, "Dreamlands", {"blue", "green", "red", "yellow"})
+  ret = events.ForceMovement(char, "House")
+  dreamlands = events.Sequence([move_dream, enc_dream, ret], char)
+  move_abyss = events.ForceMovement(char, "Abyss1")
+  # TODO: don't hard-code.
+  enc_abyss = events.GateEncounter(char, "Abyss", {"blue","red"})
+  abyss = events.Sequence([move_abyss, enc_abyss, ret], char)
+  return events.PassFail(char, luck, dreamlands, abyss)
+def House2(char):
+  move = events.ForceMovement(char, "Lodge")
+  encounter = events.Encounter(char, "Lodge", 2)
+  lodge = events.Sequence([move, encounter], char)
+  return events.BinaryChoice(char, "Enter the tunnel?", "Yes", "No", lodge, events.Nothing())
+def House3(char):
+  die = events.DiceRoll(char, 1)
+  stamina = events.Gain(char, {"stamina": die})
+  return events.Sequence([die, stamina], char)
+def House4(char):
+  stay = events.Delayed(char)
+  common = events.Nothing() # TODO: coming soon from Peter! draw and purchase
+  unique = events.Nothing() # TODO: coming soon from Peter! draw and purchase 
+  item = events.BinaryChoice(char, "Purchase a common or unique item?", "Common", "Unique", common, unique)
+  will = events.Check(char, "will", 0)
+  converse = events.PassFail(char, will, item, stay)
+  return events.BinaryChoice(char, "Converse with travelling Salesman?", "Yes", "No", converse, events.Nothing())
+def House5(char):
+  return events.Draw(char, "common", 1)
+def House6(char):
+  luck = events.Check(char, "luck", -1)
+  stamina = events.Gain(char, {"stamina": 1})
+  sanity = events.Gain(char, {"sanity": 1})
+  choice = events.BinaryChoice(char, "Gain 1 stamina or sanity?", "Stamina", "Sanity", stamina, sanity)
+  return events.PassFail(char, luck, choice, events.Nothing())
+def House7(char):
+  prereq = events.AttributePrerequisite(char, "dollars", 3, "at least")
+  pay = events.Loss(char, {"dollars": 3})
+  gain = events.SplitGain(char, "sanity", "stamina", 4)
+  stay = events.Sequence([pay, gain], char)
+  choice = events.BinaryChoice(char, "Spend $3 to spend the night?", "Yes", "No", stay, events.Nothing())
```

Comment:
Same thing - can be converted to a BinaryChoice with a prerequisite.

Reply:
done